iib/web/api_v1.py

```diff
@@ -87,6 +87,20 @@ def _should_force_overwrite():
     return should_force
 
 
+def _get_user_queue():
+    """
+    Return the name of the celery task queue mapped to the current user.
+
+    :return: queue name to be used or None if the default queue should be used
+    :rtype: str or None
+    """
+    # current_user.is_authenticated is only ever False when auth is disabled
+    if not current_user.is_authenticated:
```

Comment:
I wonder if this is required since we are calling this method from the endpoint methods which already have the `login_required` decorator. Am I missing something?

Reply:
Login is based on configuration. The docker compose environment doesn't set it, for example.